centreon/www/front_src/src/ResourceAccessManagement/atom.ts

```diff
@@ -0,0 +1,16 @@
+import { atom } from 'jotai';
+
+import { ResourceAccessRuleType } from './models';
+
+type SortOrder = 'asc' | 'desc';
+
+export const limitAtom = atom<number | undefined>(10);
+export const pageAtom = atom<number | undefined>(undefined);
+export const searchAtom = atom<string>('');
+export const sortOrderAtom = atom<SortOrder>('asc');
+export const sortFieldAtom = atom<string>('name');
```

Comment:
Be careful, atoms are global across the application. It's better to use states in this case except the search atom but rename is resourceAccessManagementSearch

Reply:
Done